### PR TITLE
Feat: Add support for fingerprinting only manifest or icons.

### PR DESCRIPTION
**Pull request by Chris3773**

### CODE REVIEW

File: tests/fingerprints-icons/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
```
Comment (doug-wade) [inline tests/fingerprints-icons/.gitignore]: Please use the top-level gitignore.

Comment (Chris3773) [reply tests/fingerprints-icons/.gitignore]: I copied the test from the existing basic test folder.

Should I remove them from all the test folders or just these tests?

Comment (Chris3773) [reply tests/fingerprints-icons/.gitignore]: I moved gitignore settings to the top-level gitignore

**Comment by doug-wade** [comment]
@Chris3773 please fix the test failures and we'll get this merged

**Pull request closed without merging**